app/src/layout/WarSystem.tsx

```
@@ -0,0 +1,1369 @@
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+import type { ColumnDefinitionType } from "@/layout/Table";
+import type { ArrayElement } from "@/utils/typeutils";
+import Confirm2 from "@/layout/Confirm2";
+
+const Map = dynamic(() => import("@/layout/Map"), { ssr: false });
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)